static/js/services/vulnerability-service.js

```diff
@@ -418,7 +417,10 @@ angular.module('quay').factory('VulnerabilityService', ['Config', 'ApiService',
 
   vulnService.getVectorOptions = function(vectorString) {
     var parts = vectorString.split(':');
-    return vulnService.NVD_VECTORS[parts[0]].values;
+    if (vulnService.NVD_VECTORS[parts[0]]) {
+      return vulnService.NVD_VECTORS[parts[0]].values;
+    }
+    return null;
   };
 
   vulnService.NVD_VECTORS = {
```

Comment:
This vectors object needs to be updated for CVSSv3: https://www.first.org/cvss/v3.1/specification-document#Vector-String

Reply:
I updated the vectors object in the latest commit. Can you please review it?